tardis/opacities/macro_atom/continuum_processes/collisional_ion_trans_prob.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+
+from tardis import constants as const
+from tardis.plasma.properties.base import (
+    TransitionProbabilitiesProperty,
+)
+from tardis.plasma.properties.continuum_processes.rates import (
+    IndexSetterMixin,
+)
+
+H = const.h.cgs.value
```

Comment:
If there will be a place to hold all the constants soon, this should be updated as well.